### PR TITLE
DM-29622: Cycle 0019 preparation work

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: cycle/docker-compose.yaml [hunk 1]
```diff
@@ -378,7 +378,7 @@ services:
         dds_community_build: ${dds_community_build_el7}
         dds_private_version: ${dds_private_version}
         dds_private_build: ${dds_private_build}
-        ts_dds: ${ts_dds_private}
+        ts_dds: ${ts_dds_private}=${ts_dds_private_conda_build}
```
Comment (mareuter) [inline cycle/docker-compose.yaml:381]: It that really an = sign or should it be a -?

File: cycle/docker-compose.yaml [hunk 2]
```diff
@@ -480,7 +480,7 @@ services:
       args:
         hub: ts-dockerhub.lsst.org
         base_image_tag: ${CYCLE}
-        ts_dds: ${ts_dds_private}
+        ts_dds: ${ts_dds_private}=${ts_dds_private_conda_build}
```
Comment (mareuter) [inline cycle/docker-compose.yaml:483]: It that really an = sign or should it be a -?

File: doc/version-history.rst [hunk 3]
```diff
@@ -8,6 +8,12 @@ Version History
 .. No new work should be required in order to complete this section.
 .. Below is an example of a version history format.
 
+Cycle 0018 Revision 002
+=======================
+
+* Update ATPtg configuration to remove restricted rotator limits.
+* Update version of atspectrograph.
```
Comment (mareuter) [inline doc/version-history.rst:15]: To be consistent, use ATSpectrograph

File: doc/version-history.rst [hunk 4]
```diff
@@ -8,6 +8,12 @@ Version History
...
+
```
Comment (mareuter) [inline doc/version-history.rst:16]: Also looks like we should add here: Initial support for the ESS.

**Review by mareuter — APPROVED** [review]
Couple of things to look at, but otherwise looks fine.

File: cycle/docker-compose.yaml (hunk 1 above)
Comment (tribeiro) [reply cycle/docker-compose.yaml:381]: it is an "=" (https://stackoverflow.com/questions/48128029/installing-specific-build-of-an-anaconda-package)

File: cycle/docker-compose.yaml (hunk 2 above)
Comment (tribeiro) [reply cycle/docker-compose.yaml:483]: yeap.. really is "=".. :)

**Pull request merged**